local-cli/server/runServer.js

```diff
@@ -88,7 +88,7 @@ function getPackagerServer(args, config) {
       'aac', 'aiff', 'caf', 'm4a', 'mp3', 'wav', // Audio formats
       'html', 'pdf', // Document formats
     ],
-    resetCache: args.resetCache || args['reset-cache'],
+    resetCache: args.resetCache,
```

Comment:
commander camel cases the flags anyway, so regardless whether it's `reset-cache` or `resetCache` that we allow to be passed, it's always accessible the same way.